CONTRIBUTING.md

```diff
@@ -1,15 +0,0 @@
-### Submitting a new package
```

Comment:
If you rebase your branch on the `new-pr` branch, these files will be gone along with unrelated commits in the history.